tools/biomoldyn/test-data/NEQGamma_outp1.txt

```diff
@@ -1,102 +1,102 @@
-     0.00014600          -0.01576224        3986.15244966           0.00000313          -0.01576537
-     0.00014800          -0.01501718        5089.68547076           0.00001221          -0.01502939
-     0.00015000          -0.01296828        3745.69661691           0.00002104          -0.01298932
-     0.00015200          -0.01017811         217.77219521           0.00002501          -0.01020312
-     0.00015400          -0.00739771       -3487.48685363           0.00002174          -0.00741945
-     0.00015600          -0.00530829       -5061.01775777           0.00001319          -0.00532148
-     0.00015800          -0.00432459       -4116.54923544           0.00000401          -0.00432860
-     0.00016000          -0.00449294       -2092.85670093          -0.00000220          -0.00449074
-     0.00016200          -0.00550412        -649.87005150          -0.00000494          -0.00549918
-     0.00016400          -0.00660743          -5.91086837          -0.00000560          -0.00660183
-     0.00016600          -0.00753781          65.54549512          -0.00000554          -0.00753227
-     0.00016800          -0.00787258          94.68015637          -0.00000538          -0.00786720
-     0.00017000          -0.00715284        -179.12143808          -0.00000546          -0.00714738
-     0.00017200          -0.00581344        -272.82554896          -0.00000591          -0.00580753
-     0.00017400          -0.00439586         355.10351308          -0.00000583          -0.00439003
-     0.00017600          -0.00348190        1657.54592740          -0.00000382          -0.00347808
-     0.00017800          -0.00349891        2884.74159668           0.00000072          -0.00349964
-     0.00018000          -0.00462675        3023.25287064           0.00000663          -0.00463339
-     0.00018200          -0.00676588        1754.81618622           0.00001141          -0.00677729
-     0.00018400          -0.00947283         -54.45952336           0.00001311          -0.00948594
-     0.00018600          -0.01198297        -944.02511479           0.00001211          -0.01199508
-     0.00018800          -0.01351134        -537.66663608           0.00001063          -0.01352197
-     0.00019000          -0.01359850         619.79872886           0.00001071          -0.01360921
-     0.00019200          -0.01230380        1830.14069868           0.00001316          -0.01231696
-     0.00019400          -0.01014874        2571.36327134           0.00001756          -0.01016630
-     0.00019600          -0.00786581        2431.86699351           0.00002257          -0.00788837
-     0.00019800          -0.00618981        1426.36740716           0.00002643          -0.00621624
-     0.00020000          -0.00567906        -123.79373740           0.00002773          -0.00570679
+#x	force_integral	frict_coeff	wdiss	corrected_force_integral
+     0.00000000	          0.00000000	          0.00000000	          0.00000000	          0.00000000
```

Comment:
@simonbray this looks like a tabular and whitespaces, is that on purpose?

Reply:
See here: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/88/files#diff-06c748dcd2176f10cd6cc2ae821c75d1d612d988503a82d7ca638aca025b43f6L135

The previous formatting was not getting recognized as a proper tabular. I hope this answers your question?